docs/audit_report/src/side_channels/02_04_sphincsplus.rst

```diff
@@ -0,0 +1,196 @@
+""""""""
+SPHINCS+
+""""""""
+
+Analysierte Variante:
+
+- SphincsPlus-sha2-128s-r3.1, deterministic & randomized
+
+Für die Analyse von SPHINCS+ wurde ein Hilfsprogramm geschrieben, das ähnlich wie das Botan CLI die zu analysierenden Funktionen aufruft.
+Für die Erzeugung der Signatur wird folgender Aufruf verwendet:
+
+.. code-block:: cpp
+
+    auto params = Botan::Sphincs_Parameters::create("SphincsPlus-sha2-128s-r3.1");
+    Botan::SphincsPlus_PrivateKey priv_key(params.algorithm_identifier(), priv_key_bits);
+
+    #if DETERMINISTIC
+        Botan::PK_Signer sig(priv_key, rng, "Deterministic");
+    #else
+        Botan::PK_Signer sig(priv_key, rng, "Randomized");
+    #endif
+        signature = sig.sign_message(message, rng);
+
+Der SPHINCS+ Standard verwendet innerhalb des Hypertrees eine Variante von XMSS und bezeichnet diese verkürzt als XMSS.
+Dem Standard folgend wird dieses Verfahren auch in diesem Abschnitt als XMSS bezeichnet.
+
+**Modifikation des Hypertrees**
+
+Die Laufzeit zur Erzeugung einer Signatur hängt von der Höhe des Hypertrees und der Anzahl der Merkle-Bäume ab.
+Um die Laufzeit zu reduzieren, wird die Gesamthöhe auf vier beschränkt und es werden zwei Merkle-Bäume mit jeweils einer Höhe von zwei verwendet.
```

Comment:
Evtl. hier anmerken, dass dies nötig ist um die Laufzeit der Analyse praktikabel zu machen. Der Algorithmus selbst benötigt diese Einschränkung im Produktiveinsatz nicht.

Reply:
Fix in https://github.com/sehlen-bsi/botan-docs/pull/113/commits/49e079bd85018e5905223d4d26769845abc7550e